src/connector.rs

```diff
@@ -115,9 +119,11 @@ impl EthConnectorContract {
 
         let msg = hex::decode(data[1]).expect(ERR_FAILED_PARSE);
         let mut fee: [u8; 32] = Default::default();
+        // TODO: Check msg length?
         fee.copy_from_slice(&msg[..32]);
         let mut recipient: EthAddress = Default::default();
         recipient.copy_from_slice(&msg[32..52]);
+        // TODO: Do you need to verify validity of account ID in the data[0]?
```

Comment:
It's 20-bytes of Eth address. Do we really need some additional checks?
In the other Aurora places, we check only slice length.